src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -198,60 +202,75 @@
+            confirm-disabled?                (or (nil? route)
+                                                 (empty? route)
+                                                 (empty? @input-value)
+                                                 (<= input-num-value 0)
+                                                 (> input-num-value (current-limit)))
+            amount-text                      (str @input-value " " token-symbol)
+            native-currency-symbol           (when-not confirm-disabled?
+                                               (get-in route [:from :native-currency-symbol]))
+            native-token                     (when native-currency-symbol
+                                               (rf/sub [:wallet/token-by-symbol native-currency-symbol]))
+            fee-in-native-token              (when-not confirm-disabled?
+                                               (send-utils/calculate-full-route-gas-fee route))
+            fee-in-crypto-formatted          (when fee-in-native-token
+                                               (utils/get-standard-crypto-format native-token
+                                                                                 fee-in-native-token))
+            fee-in-fiat                      (when-not confirm-disabled?
+                                               (utils/token-fiat-value fiat-currency
+                                                                       fee-in-native-token
+                                                                       native-token))
+            currency-symbol                  (rf/sub [:profile/currency-symbol])
+            fee-formatted                    (when fee-in-fiat
+                                               (utils/get-standard-fiat-format fee-in-crypto-formatted
+                                                                               currency-symbol
+                                                                               fee-in-fiat))
+            selected-networks                (rf/sub [:wallet/wallet-send-selected-networks])
+            affordable-networks              (send-utils/find-affordable-networks
+                                              {:balances-per-chain token-balances-per-chain
+                                               :input-value        @input-value
+                                               :selected-networks  selected-networks
+                                               :disabled-chain-ids disabled-from-chain-ids})]
         (rn/use-mount
          (fn []
            (let [dismiss-keyboard-fn   #(when (= % "active") (rn/dismiss-keyboard!))
                  app-keyboard-listener (.addEventListener rn/app-state "change" dismiss-keyboard-fn)]
              #(.remove app-keyboard-listener))))
         (rn/use-effect
-         #(fetch-routes input-num-value (current-limit))
+         #(when (> (count affordable-networks) 0)
```

Comment:
do we really need two use-effects?

Reply:
I initially reused the same useEffect, but then I found we need different bounce durations when changing disabled networks than when changing the input amount.